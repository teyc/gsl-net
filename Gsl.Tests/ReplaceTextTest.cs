using Xunit;

namespace Gsl.Tests
{
    public class ReplaceTextTest : TestBase
    {
        public ReplaceTextTest(Xunit.Abstractions.ITestOutputHelper log) : base(log)
        {
        }

        [Fact]
        public void TemplateTextShouldBeReplaced()
        {
            const string template = "data/replaceText.gsl";
            const string data = "data/replaceText.json";

            TemplateWithSingleFileOutput(template, data);
        }
    }
}