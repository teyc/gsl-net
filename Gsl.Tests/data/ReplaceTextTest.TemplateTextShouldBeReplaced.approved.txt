file: replaceText.Baz.cs
namespace Gsl.Tests
{
    public class Baz
    {
        public static void CreateBaz() { };
        public static void UpdateBaz(Baz item) { };
        public static void DeleteBaz(int id) { };
    }
}
file: replaceText.Bar.cs
namespace Gsl.Tests
{
    public class Bar
    {
        public static void CreateBar() { };
        public static void UpdateBar(Bar item) { };
        public static void DeleteBar(int id) { };
    }
}
file: replaceText.Quux.cs
namespace Gsl.Tests
{
    public class Quux
    {
        public static void CreateQuux() { };
        public static void UpdateQuux(Quux item) { };
        public static void DeleteQuux(int id) { };
    }
}